### ISSUE TITLE
DeprecationWarning: loaderUtils.parseQuery()

### ISSUE THREAD

**Issue by 3axap4eHko** [MoOx/eslint-loader#161]
The warning message:
``` javascript
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```
The stack trace of warning message:
``` javascript
    at Object.parseQuery (node_modules/loader-utils/index.js:79:15)
    at Object.module.exports (node_modules/babel-loader/lib/index.js:96:35)
    at LOADER_EXECUTION (node_modules/loader-runner/lib/LoaderRunner.js:119:14)
    at runSyncOrAsync (node_modules/loader-runner/lib/LoaderRunner.js:120:4)
    at iterateNormalLoaders (node_modules/loader-runner/lib/LoaderRunner.js:229:2)
    at iterateNormalLoaders (node_modules/loader-runner/lib/LoaderRunner.js:218:10)
    at node_modules/loader-runner/lib/LoaderRunner.js:233:3
    at Object.context.callback (node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.module.exports (node_modules/eslint-loader/index.js:198:8)
```
suppose this warning belongs to eslint-loader

**Comment by NTARelix** [comment]
See #156. This was resolved in commit 82fae058d9bdac76e8ffd3d712201e0ae07a233f. Updating `eslint-loader` to the latest version (currently `1.6.3`) should solve this problem.
